packages/trie/src/trie/trie.ts

```diff
@@ -942,8 +942,8 @@ export class Trie {
    * Returns a copy of the underlying trie.
    * @param includeCheckpoints - If true and during a checkpoint, the copy will contain the checkpointing metadata and will use the same scratch as underlying db.
    */
-  async copy(includeCheckpoints = true): Promise<Trie> {
-    const trie = await Trie.create({
+  copy(includeCheckpoints = true): Trie {
+    const trie = new Trie({
```

Comment:
Ah, please, as mentioned, do not revert this. This was a great change! 🤩

Reply:
oh -- sure.  even though it makes this PR breaking?

Reply:
ah, just saw your comment above

Reply:
Yes, now it's the time to make these kind of breaking changes if they are useful. 🙂